doc: update `-manager-addr` doc (#3946)
juicedata/juicefs: docs/en/reference/command_reference.md

#### Cluster related options {#sync-cluster-related-options}

|Items| Description                                                                                                                                                                                                                       |
|-|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> |`--manager-addr=ADDR`| Listening address of the Manager node in distributed synchronization mode in the format: `<IP>:[port]`, or a random port if no port is written. If no port is written, it listens on a random local ipv4 address and a random port. |

Reviewer: ```suggestion
|`--manager-addr=ADDR`| The listening address of the Manager node in distributed synchronization mode in the format: `<IP>:[port]`. If not specified, it listens on a random port. If this option is omitted, it listens on a random local IPv4 address and a random port. |
```
